libpf/pfelf/file.go

```diff
@@ -537,13 +570,23 @@ func (f *File) OpenDebugLink(elfFilePath string, elfOpener ELFOpener) (
 	linkName, linkCRC32, err := f.GetDebugLink()
 	if err != nil {
 		// Treat missing or corrupt tag as soft error.
-		return
+		return nil, ""
 	}
 
 	// Try to find the debug file
 	executablePath := filepath.Dir(elfFilePath)
-	for _, debugPath := range []string{"/usr/lib/debug/"} {
-		debugFile = filepath.Join(debugPath, executablePath, linkName)
+
+	debugDirectories := []string{
+		executablePath,
+		filepath.Join(executablePath, ".debug"),
+	}
+	for _, dir := range globalDebugDirectories {
+		debugDirectories = append(debugDirectories,
+			filepath.Join(dir, filepath.Base(executablePath)))
```

Comment:
```suggestion
			filepath.Join(dir, executablePath))
```

From [gdb doc](https://sourceware.org/gdb/current/onlinedocs/gdb.html/Separate-Debug-Files.html):
> For the “debug link” method, GDB looks up the named file in the directory of the executable file, then in a subdirectory of that directory named .debug, and finally under each one of the global debug directories, in a subdirectory whose name is identical to the leading directories of the executable’s absolute file name.